_rules/audio-transcript-2eb176.md

```diff
@@ -5,6 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that `audio` elements have a transcript that includes all auditory information.
 accessibility_requirements:
+  wcag20:1.2.1: # Audio-only and Video-only (Prerecorded) (A)
+    forConformance: true
+    secondary: This success criterion is **less strict** than this rule. This is because the rule does not consider that the audio may be a media alternative for text. Some of the failed examples satisfy this success criterion.
```

Comment:
```suggestion
  wcag20:1.2.1: # Audio-only and Video-only (Prerecorded) (A)
    secondary: This success criterion is **less strict** than this rule. This is because the rule does not consider that the audio may be a media alternative for text. Some of the failed examples satisfy this success criterion.
```